plasmapy/atomic/particle_input.py

```diff
@@ -242,7 +242,7 @@ def wrapper(*args, **kwargs):
                     expected_params = len(annotated_argnames)
                     received_params = len(arguments[argname])
                     if not expected_params == received_params:
-                        raise TypeError(
+                        raise ValueError(
```

Comment:
This will probably take an update in the docstring

Reply:
Okay. I pushed some changes. Let me know if we need changes in the wording.